PWG/EMCAL/EMCALtasks/AliAnalysisTaskEmcalTriggerNormalization.cxx

```diff
@@ -206,29 +205,6 @@ Bool_t AliAnalysisTaskEmcalTriggerNormalization::Run(){
           auto triggerweight = emcaltrigger.second * mbweight;
           luminosityHist->Fill(luminosityHist->GetXaxis()->GetBinCenter(triggerbin), centralitypercentile, triggerweight);
         }
-
-        // fill luminosity
-        for(auto edtrigger : mixedtriggers) {
-          std::string emcaltriggerclass = "E" + edtrigger.substr(2),
-                      dcaltriggerclass = "D" + edtrigger.substr(2);
-          auto emcaltriggerEnabled = fCacheTriggerClasses.find(emcaltriggerclass),
-               dcaltriggerEnabled = fCacheTriggerClasses.find(dcaltriggerclass);
-          if((emcaltriggerEnabled != fCacheTriggerClasses.end()) || (dcaltriggerEnabled != fCacheTriggerClasses.end())) {
-            // Either EMCAL or DCAL trigger enabled, get downscale factor
-            // If EMCAL trigger enabled perfer this (covers practically all cases)
-            double downscaleTrigger = 0.;
-            if(emcaltriggerEnabled != fCacheTriggerClasses.end()) {
-              downscaleTrigger = fCacheDownscaleFactors[emcaltriggerclass];
-            } else {
-              // Only DCAL triggers would enabled
-              // this should never happen
-              downscaleTrigger = fCacheDownscaleFactors[dcaltriggerclass];
-            }
-            auto triggerbin = luminosityHist->GetXaxis()->FindBin(edtrigger.data());
-            auto triggerweight = downscaleTrigger * mbweight;
-            luminosityHist->Fill(luminosityHist->GetXaxis()->GetBinCenter(triggerbin), centralitypercentile, triggerweight);
-          }
-        }
```

Comment:
Loop no longer needed because the luminosity histogram is alley filled for the mixed triggers in the previous loop over all cached triggers.